.idea/bucketlist.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+  <component name="TestRunnerService">
+    <option name="PROJECT_TEST_RUNNER" value="Unittests" />
+  </component>
+</module>
```

Comment:
Please add this file (.idea/ ) to .gitignore

Reply:
I added it but it still doesn't ignore idea files. I don't seem to understand why

Reply:
That's because you've added it to revision control once so git will always track it.
To fix, delete and commit the delete.
pycharm will re-create the hidden file on it's one (ensure that when you check git status .idea/ is not part of the files you'll be committing )